### PR TITLE
Add compat data for SVG <pattern>

### PR DESCRIPTION
**Pull request by JeremiePat**
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/pattern

### CODE REVIEW

File: svg/elements/pattern.json [hunk 1]
```diff
@@ -0,0 +1,478 @@
+{
+  "svg": {
+    "elements": {
+      "mask": {
```
Comment (SebastianZ) [inline svg/elements/pattern.json]: Needs to be `pattern`.

File: svg/elements/pattern.json [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Element/mask",
```
Comment (SebastianZ) [inline svg/elements/pattern.json]: Again `pattern`.

File: svg/elements/pattern.json [hunk 3]
```diff
@@ -0,0 +1,478 @@
...
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "patternUnits": {
```
Comment (SebastianZ) [inline svg/elements/pattern.json]: `patternUnits` might not be supported in all browsers yet, meaning browsers could fall back to the default value for it.

File: svg/elements/pattern.json [hunk 4]
```diff
@@ -0,0 +1,478 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "patternContentUnits": {
```
Comment (SebastianZ) [inline svg/elements/pattern.json]: Like `patternUnits` not all browsers may support this attribute.

File: svg/elements/pattern.json [hunk 5]
```diff
@@ -0,0 +1,478 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "patternTransform": {
```
Comment (SebastianZ) [inline svg/elements/pattern.json]: Like `patternUnits` not all browsers may support this attribute.

File: svg/elements/pattern.json [hunk 6]
```diff
@@ -0,0 +1,478 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "href": {
```
Comment (SebastianZ) [inline svg/elements/pattern.json]: Not sure all browsers already support `href` in addition to `xlink:href`, as that's a change in SVG 2.

**Comment by JeremiePat** [comment]
All fixed :)

**Pull request merged**